scap-workbench-oscap.sh

```diff
@@ -74,7 +74,7 @@ while kill -0 $PID 2> /dev/null; do
     ret=$?
     if [ 0 -lt $ret -a $ret -lt 128 ]; then
         # If read failed & it was not due to timeout --> parents are gone.
-        kill -s SIGINT $PID 2> /dev/null
+        kill -TERM $PID 2> /dev/null
```

Comment:
We shouldn't just TERM the process, that will leave dangling resources. Is "TERM" really needed there?